.github/workflows/release.yml

```diff
@@ -3,7 +3,7 @@ name: Publish Release
 on:
   workflow_dispatch:
   repository_dispatch:
-    types: [ connector-release-pipeline ]
+    types: [ stdlib-release-pipeline ]
```

Comment:
is this change intentional?

Reply:
No, I think I got this from GitHub, will change,